test/new_relic/agent/rpm_agent_test.rb

```diff
@@ -46,7 +46,7 @@ def test_startup_shutdown_real
   def test_manual_start
     NewRelic::Agent.instance.expects(:connect).once
     NewRelic::Agent.instance.expects(:start_worker_thread).once
-    NewRelic::Agent.instance.instance_variable_set '@started', nil
+    NewRelic::Agent.instance.instance_variable_set :@started, nil
```

Comment:
The use of a symbol instead of a string here is a good change!

For the very specific situation of wanting to blank out a variable as we're doing here, I think `remove_instance_variable(:@variable_name)` would be even better than `instance_variable_set(:@variable_name, nil)`.

Reply:
Thank you! As chatted about, will explore making this nifty change across codebase in separate PR 🌈